canonical-web-and-design/maas-ui
# refactor(ui): remove old action dispatches

## Done

- Remove the dispatches for the old action types.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Open the redux dev tools and load the UI.
- You should only see dispatches in the format `model/actionName` and not `MODEL_ACTION_NAME`.

## Fixes

Fixes: #1426.
